pkg/controller/controller.go

```diff
@@ -163,9 +167,10 @@ func (m *CertificateApprover) reconcileCSR(csr certificatesv1.CertificateSigning
 		klog.Errorf("failed to get kubelet CA")
 	}
 
-	if err := authorizeCSR(m, m.Config, machines.Items, &csr, parsedCSR, kubeletCA); err != nil {
+	if authorize, err := authorizeCSR(m, m.Config, machines.Items, &csr, parsedCSR, kubeletCA); !authorize {
 		// Don't deny since it might be someone else's CSR
-		return fmt.Errorf("CSR %s not authorized: %v", csr.Name, err)
+		klog.Infof("%s: CSR not authorized", csr.Name)
+		return err
```

Comment:
Will the `err` not contain any useful information?

Reply:
The error will have already been logged inside the function.